Plugin/Tax/Config/ShippingTaxPlugin.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FireGento\MageSetup\Plugin\Tax\Config;
+
+use FireGento\MageSetup\Model\System\Config\Source\Tax\Dynamic as FireGentoSource;
```

Comment:
I would prefer a more specific alias like `DynamicTaxSource`, but fine for me :)

Reply:
this source model was removed, instead magento code `Yesno` source is used.